### DISCUSSION TITLE
Ideas on how to create a table`s column with a select to change the status attribute for the Proposal Model.

### DISCUSSION THREAD

**Discussion by otavio-araujo** [filamentphp/filament#2850] [Help] [answered]
Does anyone has any idea on how to make a custom column that has a select to change a Model`s attribute directly inside the table on list page?

**Comment by danharrin** [comment, marked as answer]
You could use the `wire:change` attribute on the `<select>` to call a Livewire method on the list page, and update the value in the database?

**Reply by otavio-araujo** [reply]
Thanks very much again!
